### DISCUSSION TITLE
Rspamd integration

### DISCUSSION THREAD

**Discussion by Skywrath-Mage** [postalserver/postal#1684] [Help with using Postal]
Hello, 

After 2.1 postal support Rspamd, a good improvement over SpamAssasin, but the documentation is still missing and even if I was able to connect the 2 I add not such an easy task. 

The only thing that I was not able to fix is the:
```
HFILTER_HOSTNAME_UNKNOWN
Unknown client hostname (PTR or FCrDNS verification failed)

```
I test the same rspamd with Haraka MTA, and this issue was not there, is possible that is related to postal?
If not, do you have any suggestions on how to fix it?

**Comment by willpower232** [comment]
It sounds like rspamd is unable to look up the DNS value for the domain in the email address.

You can try using `nslookup` on the server running rspamd and/or Postal to see if it can find the value for the DNS record (I'm not sure how to do reverse DNS off the top of my head)

**Reply by Skywrath-Mage** [reply]
that's the thing, I already test and it's working fine with net-tools I can look for the PTR or DNS entry. 

And with Haraka we didn't have this issue, so it's probably something that doesn't work with postfix or postal + RspamD.

For the label I have set up: none, should I set up something particular?

**Reply by Skywrath-Mage** [reply]
the only thing I found is this public mail, with some details on postfix not passing the FCrDNS to RspamD. 
My german is not good. 
https://listi.jpberlin.de/pipermail/postfixbuch-users/2019-March/066489.html
http://www.postfix.org/MILTER_README.html

So it could not be not a solution.
